test/e2e/antctl_test.go

```diff
@@ -61,7 +70,16 @@ func TestAntctlAgentLocalAccess(t *testing.T) {
 		t.Fatalf("Error when getting antrea-agent pod name: %v", err)
 	}
 	for _, c := range antctl.CommandList.GetDebugCommands(runtime.ModeAgent) {
-		args := append([]string{"antctl", "-v"}, c...)
+		args := []string{}
+		if testOptions.enableCoverage {
+			const timeFormat = "20060102T150405Z0700"
+			timeStamp := time.Now().Format(timeFormat)
+			args = append([]string{"antctl-coverage", "-test.run=TestBincoverRunMain", fmt.Sprintf("-test.coverprofile=antctl-%s.out", timeStamp)}, c...)
+		} else {
+			args = append([]string{"antctl", "-v"}, c...)
+		}
+		t.Logf("args: %s", args)
+
```

Comment:
what about for `TestAntctlControllerRemoteAccess`?

Reply:
Also changed the TestAntctlControllerRemoteAccess func to measure antctl coverage, then copy the antctl coverage file from master node to the coverage directory.

TestAntctlVerboseMode cannot be changed since  "-v" option has some other meaning for "antctl-coverage" binary. 

  -v value
        number for the log level verbosity